### DISCUSSION TITLE
Change color of the bar based on a column on Superset

### DISCUSSION THREAD

**Discussion by beruz** [apache/superset#15009] [Q&A / Help]
Hi, its been a month since I started using Apache Superset. I have a graph that gives the accuracy of an event occuring on a given hour shown below.

![image](https://user-images.githubusercontent.com/44114094/120929096-5ecee480-c6f0-11eb-8993-ab0f3115fd9e.png)

The event occuring is a column named 'prediction' which is either 'Left' or 'Right'. Is there a way to colorize the bars according to the value in 'prediction' column. If the prediction is Left then red colored bar and green colored bar when Right etc. I have checked metadata but I am not sure how I can edit it. Is there a way to achieve this?

**Comment by kamalkeshavani-aiinside** [comment]
So adding 'prediction' column in Breakdown section of query is not what you expect?
